Assets/MixedRealityToolkit.SDK/Experimental/Features/UX/KeyboardInputFieldBase.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine.UI;
+#if WINDOWS_UWP
+using UnityEngine.EventSystems;
+using Microsoft.MixedReality.Toolkit.Input;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI
+{
+    public abstract class KeyboardInputFieldBase<T> : MixedRealityKeyboardBase<T>
+#if WINDOWS_UWP
```

Comment:
Does this only work for .NET core? I don't believe WINDOW_UWP is the right preprocessor

I think you just want UNITY_WSA if just targeting UWP app platform
https://docs.unity3d.com/Manual/PlatformDependentCompilation.html

Reply:
The MRTK documentation says that it's equivalent to !UNITY_EDITOR && UNITY_WSA, which is where I wanted this to work.
https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html

Reply:
Interesting, never heard of that before